pex/tools/command.py

```diff
@@ -120,17 +128,29 @@ def show_help(
         # type: (...) -> NoReturn
         parser.error("a subcommand is required")
 
-    def add_arguments(self, parser):
+    @classmethod
+    def name(cls):
+        # type: () -> str
+        return cls.__name__.lower()
+
+    @classmethod
+    def description(cls):
+        # type: () -> Optional[str]
+        return cls.__doc__
+
+    @classmethod
+    def add_arguments(cls, parser):
         # type: (ArgumentParser) -> None
         pass
 
-    def run(
-        self,
-        pex,  # type: PEX
-        options,  # type: Namespace
-    ):
-        # type: (...) -> Result
-        pass
+    options = attr.ib()  # type: Namespace
+
+
+class PEXCommand(Command):
```

Comment:
Everything in this file, except this here PEXCommand will be able to be moved wholesale to the `pex.commands` package for re-use by the `pex3` cli. I'll do that pure move in a follow-up with a minimal pex3 CLI that just supports `-h`/`-V` to start.

Reply:
pex3? I know we discussed that in meeting, but I thought you'll be using PEX_TOOLS for now. Is that in the roadmap for near future?